gmd.py

```
@@ -322,6 +542,611 @@ def interactive_menu(env: CommandEnv) -> None:
+            return
+        selected_paths = selection
+
+    result = sync_claude_code_sessions(
+        base_dir=base_dir,
+        output_dir=out_dir,
+        collapse_threshold=collapse,
+        html=html_enabled,
+        html_theme=html_theme,
+        force=force,
+        prune=prune,
+        diff=diff_enabled,
```

Ruff (0.15.9)

[error] Undefined name `diff_enabled`

(F821)